components/admin-panel/Menu.js

```diff
@@ -74,7 +75,7 @@ const Menu = ({ collective }) => {
           </SubMenu>
           <SubMenu
             label={<FormattedMessage id="AdminPanel.FiscalHostSettings" defaultMessage="Fiscal Host Settings" />}
-            if={isHost || (isType(collective, USER) && isHost)}
+            if={isHost || (isIndividual && isHost)}
```

Comment:
If I read the code, seems `isHost` should be enough?

Reply:
Definitely. I did a simple search & replace on `isType(collective, USER)`, did not notice that the condition was redundant